bencyn/ah-console-mini-app
# #export article

#### Title
Export an article

#### Description

create  a cli command that exports an article by its slug name from Authors Haven to a local json file

#### Type of change






#### How Has This Been Tested?
-  clone the repository 
    `git clone https://github.com/bencyn/ah-console-mini-app.git`

-  check out the branch 
   `feature/166570682-fix-login`

- create a virtual environment
  `virtualenv --python python3 venv`

- install the app
   `pip install --editable .`

-  run export command  
   `ah export <article_slug>`.

#### Checklist:
- [x] Create an export command
- [x] Create an exports article directory

#### Screenshots

![image](https://user-images.githubusercontent.com/11487123/59745382-5966fe00-927d-11e9-9cc5-65425cec20b1.png)

-- export file 

![image](https://user-images.githubusercontent.com/11487123/59745433-7a2f5380-927d-11e9-9318-10dfe476392b.png)
